Assets/Materials/Gate/New Material 1.mat

```diff
@@ -0,0 +1,133 @@
+%YAML 1.1
```

Comment:
Not sure what this material is, should be renamed (or maybe deleted if it's not used)

Reply:
deleted material